bin/tape

```diff
@@ -1,12 +1,33 @@
 #!/usr/bin/env node
 
-var path = require('path');
+var resolveModule = require('resolve').sync;
+var resolvePath = require('path').resolve;
+var parseOpts = require('minimist');
 var glob = require('glob');
 
-process.argv.slice(2).forEach(function (arg) {
+var opts = parseOpts(process.argv.slice(2), {
+        alias: { 'r': 'require' },
+        string: 'require',
+        default: {'r': [] }
+    });
+
+var cwd = process.cwd();
+
+if (typeof opts.require === 'string') {
+  opts.require = [opts.require];
+}
+
+opts.require.forEach(function(module) {
+    /* The `module &&` ensures we ignore `-r ""`, trailing `-r` or other
+     * silly things the user might (inadvertedly) be doing.
+     */
+    module && require(resolveModule(module, { basedir: cwd }));
```

Comment:
i'd prefer `if (module) { require(resolveModule(module, { basedir: cwd })); }`, rather than abusing boolean operators for control flow :-/